packages/clerk-js/src/core/clerk.ts

```diff
@@ -1206,9 +1206,10 @@ export class Clerk implements ClerkInterface {
 
   #shouldSyncWithPrimary = (): boolean => {
     if (this.#hasJustSynced()) {
-      if (!this.#options.isInterstitial) {
-        this.#clearJustSynced();
-      }
+      // TODO @nikos
+      // if (!this.#options.isInterstitial) {
+      //   this.#clearJustSynced();
+      // }
```

Comment:
I _think_ we can do this:

```suggestion
      this.#clearJustSynced();
```